### DISCUSSION TITLE
Outlook email unable to send mail

### DISCUSSION THREAD

**Discussion by rduToNature** [emailjs-com/help#48] [Q&A] [answered]
The code is working fine with sending emails when using gmail, however when switched to businese outlook mail it is unable to send mail via auto reply. 

This is the error i get

Remote server returned '550 5.7.708 Service unavailable. Access denied, traffic not accepted from this IP. For more information please go to http://go.microsoft.com/fwlink/?LinkId=526653 AS(8561) [YT2P288MB0002.CANP288.PROD.OUTLOOK.COM 2023-09-08T00:17:44.250Z 08DBAE7DE5230C85]'

The mail is blocked by both gmail and other outlook mail in testing.

**Comment by xr0master** [comment]
Hi, please note, that you are getting this error from Outlook. Which version of Outlook service are you using?

**Reply by rduToNature** [reply]
Should be the latest as I use a recently purchased web version,

**Reply by rduToNature** [reply, marked as answer]
problem is fixed on its own, probablly encountered network issue
